src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,30 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
     private String title;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Using FetchType.EAGER for the 'actors' collection may lead to performance issues. Eager fetching loads the associated collection immediately, even if it's not needed, which can be inefficient for large collections. Consider using FetchType.LAZY to load the data on demand.